buildscripts/condarecipe.local/meta.yaml

```diff
@@ -34,18 +34,18 @@ requirements:
     - setuptools
     - importlib_metadata       # [py<39]
     # On channel https://anaconda.org/numba/
-    - llvmlite >=0.40.0dev0,<0.40
```

Comment:
Manual `conda build` and then install + check test suite starts suggests this is ok.